### PR TITLE
splash screen 3 sec & login page

### PR DESCRIPTION
**Pull request by cmadusankahw**
I've designed splash screen  with 3 sec timer included in it's code.
And also includes the login activity.

### CODE REVIEW

File: app/src/main/java/com/tecOps/workflow/view/login_page.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import com.kalpa.workflow.R;
+
+public class login_page extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login_page);
+
+        TextView textView2=(TextView) findViewById(R.id.textView2);
```
Comment (ahmedifhaam) [inline app/src/main/java/com/tecOps/workflow/view/login_page.java]: meaning full names would be better right ?

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,19 +1,38 @@
...
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="1.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.333"
+        app:srcCompat="@drawable/res2" />
 
+    <ImageView
+        android:id="@+id/imageView4"
+        android:layout_width="107dp"
```
Comment (ahmedifhaam) [inline app/src/main/res/layout/activity_main.xml]: hard coded sizes ? @cmadusankahw @kalpawijesooriya

File: app/src/main/res/layout/activity_login_page.xml [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".view.login_page">
+
+    <ImageView
+        android:id="@+id/background"
+        android:layout_width="503dp"
+        android:layout_height="1243dp"
```
Comment (kalpawijesooriya) [inline app/src/main/res/layout/activity_login_page.xml]: Dot hardcore width and height use match_parent and wrap_content

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (kalpawijesooriya) [reply app/src/main/res/layout/activity_main.xml]: @cmadusankahw  use 'wrap_content' or  ,match_parent' for height and width. Plz correct that in every element ( images layouts)
screenshot_2019-02-15-14-41-06
).

File: app/src/main/java/com/tecOps/workflow/view/login_page.java (hunk 1 above)
Comment (kalpawijesooriya) [reply app/src/main/java/com/tecOps/workflow/view/login_page.java]: Yah.. 
@cmadusankahw  malli we dont need to initialize elements in View. We are going to Use Data Binding neh. And use meaning full name for Image resources.

File: app/src/main/res/layout/activity_login_page.xml [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+        android:background="@null"
+        android:ems="10"
+        android:inputType="textWebEmailAddress|textPersonName"
+        android:singleLine="false"
+        android:text="Username"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.51" />
```
Comment (kalpawijesooriya) [inline app/src/main/res/layout/activity_login_page.xml]: @cmadusankahw Put 'Edit Text' inside the 'android.support.design.widget.TextInputLayout'

**Comment by kalpawijesooriya** [comment]
@cmadusankahw  malli use drawable-hdpi,drawable-mdpi,drawable-xxhdpi....... folders for put images.

File: app/src/main/java/com/tecOps/workflow/view/login_page.java (hunk 1 above)
Comment (cmadusankahw) [reply app/src/main/java/com/tecOps/workflow/view/login_page.java]: Ok ayiye. I'll correct it.

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (cmadusankahw) [reply app/src/main/res/layout/activity_main.xml]: I couldn't check that.I'll correct it ayiye.

**Pull request merged**